ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/ExecutionEngineChannel.java

```diff
@@ -25,6 +25,11 @@
 
 public interface ExecutionEngineChannel extends ChannelInterface {
 
+  enum ConsensusValidationResult {
```

Comment:
I think here we should agree on a coherent naming convention. I used `ExecutionPayloadStatus` for `engine_executePayload`. (status has been inspired by the field in the API response)

Reply:
I don't think this is too important. `ConsensusValidationStatus` works for me as well, but for some reason I like it less :)